src/run/next.cts

```diff
@@ -96,9 +104,16 @@ export async function getMockedRequestHandler(...args: Parameters<typeof getRequ
           const relPath = relative(resolve('.next/server/pages'), path)
           const file = await cacheStore.get<HtmlBlob>(relPath, 'staticHtml.get')
           if (file !== null) {
-            if (!file.isFallback) {
+            if (file.isFullyStaticPage) {
```

Comment:
isFullyStaticPage will be false for fallbacks, but also additionally it somewhat inverts the check to only set `usedFsReadForNonFallback` for KNOWN static pages instead of not setting for fallbacks (but still setting it for everything else even if it's not pages router page)